.gitlab/benchmarks/macrobenchmarks.yml

```diff
@@ -25,6 +34,42 @@ check_azure_pipeline:
   rules:
     - when: on_success
 
+update-bp-infra:
```

Comment:
Just so that I understand @faydef - when we next want to update the infra, we need to:
- Create a PR similar to [this one](https://github.com/DataDog/benchmarking-platform/pull/151) for the macro benchmarks
- Create a PR similar to [this one](https://github.com/DataDog/benchmarking-platform/pull/152) for the micro benchmarks
- Update the `MACROBENCHMARKS_CI_IMAGE` reference
- Manually run both of the `update-bp-infra` jobs

Is that correct?

Reply:
Resolving this one as discussed on Slack